### PR TITLE
data generation

### PR DESCRIPTION
**Pull request by derrickdbrandon**

### CODE REVIEW

**Comment by derrickdbrandon** [comment]
@RebeccaSawyer @NamVo5009 Could one of you review this when you get a chance? Thanks!

File: dataGenerator.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+setTimeout(makeDataFile, 20000);
+setTimeout(makeDataFile, 40000);
+setTimeout(makeDataFile, 80000);
+setTimeout(makeDataFile, 100000);
+setTimeout(makeDataFile, 120000);
+setTimeout(makeDataFile, 140000);
+setTimeout(makeDataFile, 160000);
+setTimeout(makeDataFile, 180000);
+setTimeout(makeDataFile, 200000);
+
```
Comment (rebeccasw) [inline dataGenerator.js:78]: Good idea to use setTimeout!

**Review by rebeccasw — APPROVED** [review]

File: dataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+const faker = require('faker');
+const fs = require('fs');
+
+let movieData = [];
+let fileCount = 1;
+
+let trailerLinks = ['https://www.youtube.com/watch?v=Rp8WL621uGM','https://www.youtube.com/watch?v=WKWFr9upIuU&t=15s','https://www.youtube.com/watch?v=XKB5h2tJQHQ','https://www.youtube.com/watch?v=ZVANQheoRUw','https://www.youtube.com/watch?v=OTd_D7QC8jM&t=17s','https://www.youtube.com/watch?v=WMdBs7uTLHw','https://www.youtube.com/watch?v=adzYW5DZoWs','https://www.youtube.com/watch?v=DltrMStqr8Q','https://www.youtube.com/watch?v=qeau7UWjEms','https://www.youtube.com/watch?v=xRc3WviXk2M','https://www.youtube.com/watch?v=Z6yTXsCauIc','https://www.youtube.com/watch?v=AD10dDdEGmU','https://www.youtube.com/watch?v=mEUsRi74lcY','https://www.youtube.com/watch?v=LiwFtzuSnkE','https://www.youtube.com/watch?v=dFGYK-iRMgw','https://www.youtube.com/watch?v=Hld-7oBn3Rk','https://www.youtube.com/watch?v=6IO8mjyWbhA','https://www.youtube.com/watch?v=i2Ozn_ox_4A','https://www.youtube.com/watch?v=B12REtkl2KY','https://www.youtube.com/watch?v=pU8-7BX9uxs',
```
Comment (therobinkim) [inline dataGenerator.js:7]: these links are a big unwieldy... would recommend putting them into a separate `json` file that you import into `dataGenerator.js`

File: dataGenerator.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+setTimeout(makeDataFile, 0);
+setTimeout(makeDataFile, 20000);
+setTimeout(makeDataFile, 40000);
+setTimeout(makeDataFile, 80000);
+setTimeout(makeDataFile, 100000);
+setTimeout(makeDataFile, 120000);
+setTimeout(makeDataFile, 140000);
+setTimeout(makeDataFile, 160000);
+setTimeout(makeDataFile, 180000);
+setTimeout(makeDataFile, 200000);
```
Comment (therobinkim) [inline dataGenerator.js:77]: would look into alternate way of managing async code asap

**Pull request merged**